serverless/functions/serverless-functions-annotations.adoc

```diff
@@ -0,0 +1,19 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-attributes"]
+= Adding annotations to functions
+include::modules/common-attributes.adoc[]
+:context: serverless-functions-annotations
+
+toc::[]
+
+You can add annotations to serverless functions using the `func.yaml` configuration file.
+
+[IMPORTANT]
+====
+There are two limitations of the function annotation feature:
+
+* Once a function annotation propagates to the corresponding service on the cluster, it cannot be removed from the service by deleting it from the `func.yaml` file.
+* You cannot set annotations that are set by Knative, for example, the `autoscaling` annotations.
```

Comment:
```suggestion
* You cannot override annotations that are set by Knative, for example, the `autoscaling` annotations.
```
suggestion

Reply:
@abrennan89 I'd rather leave the current wording, because override implies that Knative has already' set the annotation in question, which might not be the case.